src/manager/frontend/src/components/sections/createAgent.tsx

```diff
@@ -0,0 +1,102 @@
+import { Dialog, DialogContent, DialogFooter, DialogHeader, DialogTitle, DialogTrigger } from "../ui/dialog";
+import { Input } from "@/components/ui/input"
+import { useForm, UseFormReturn } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod"
+import { Form, FormControl, FormDescription, FormField, FormItem, FormLabel, FormMessage } from "@/components/ui/form";
+import { DialogDescription } from "@radix-ui/react-dialog";
+import { defaultRAGAppFormValues, ragappFormSchema, RAGAppFormType } from "@/client/models/service";
+import { createRAGAppService } from "@/client/service";
+import { Button } from "../ui/button";
+
+function CreateAgentForm({
+    form
+}: {
+    form: UseFormReturn<RAGAppFormType>
+}) {
+    return (
+        <Form {...form}>
+            <form
+                onSubmit={form.handleSubmit(async (data) => {
+                    await createRAGAppService(data);
+                })}
+                className="space-y-4"
+            >
+                <FormField
+                    control={form.control}
+                    name="name"
+                    render={({ field }) => (
+                        <FormItem>
+                            <FormLabel>Agent name</FormLabel>
+                            <FormControl>
+                                <Input {...field} />
+                            </FormControl>
+                            <FormDescription>
+                                Specify the name of the agent
+                            </FormDescription>
+                            <FormMessage />
+                        </FormItem>
+                    )}
+                />
+                <FormField
+                    control={form.control}
+                    name="image"
+                    render={({ field }) => (
+                        <FormItem>
+                            <FormLabel>Agent image</FormLabel>
+                            <FormControl>
+                                <Input {...field} />
+                            </FormControl>
+                            <FormDescription>
+                                Specify docker image for the agent, recommended to use the default image
+                            </FormDescription>
+                            <FormMessage />
+                        </FormItem>
+                    )}
+                />
```

Comment:
i think that's to technical for the UI - put it in the ENV variable?

Reply:
Yes, i think we can just use the default image. So generally, user just need specify app name to create a new app.